applications/plot_layout_array.py

```diff
@@ -0,0 +1,182 @@
+        required=False,
+        default=False,
+    )
+    input_group = config.parser.add_mutually_exclusive_group()
+    input_group.add_argument(
+        "--telescope_list",
+        help="ECSV file with the list of telescopes",
+        nargs="+",
+        type=str,
+        required=False,
+        default=None,
+    )
+    input_group.add_argument(
+        "--layout_array_name",
+        help="Name of the layout array.",
+        nargs="+",
+        type=str,
+        required=False,
+        default=None,
+    )
+
+    return config.initialize(db_config=True)
+
+
+def main():
+
+    label = Path(__file__).stem
+    description = "Plots layout array."
+    usage = "python applications/plot_layout_array.py --layout_array_name test_layout"
+    args_dict, _ = _parse(label, description, usage)
+    io_handler_instance = io_handler.IOHandler()
+
+    if args_dict["rotate_angle"] is None:
+        rotate_angles = [0 * u.deg]
+    else:
+        rotate_angles = []
+        for _, one_angle in enumerate(args_dict["rotate_angle"]):
+            rotate_angles.append(float(one_angle) * u.deg)
+
+    logger = logging.getLogger()
+    logger.setLevel(gen.get_log_level_from_user(args_dict["log_level"]))
+
+    if args_dict["telescope_list"] is not None:
+        logger.info("Plotting array from telescope list file(s).")
+        telescope_file = args_dict["telescope_list"]
+
+    elif args_dict["layout_array_name"] is not None:
+        logger.info("Plotting array from layout array name(s).")
+        telescope_file = [
+            io_handler_instance.get_input_data_file(
+                "layout", f"telescope_positions-{one_array}.ecsv"
+            )
+            for _, one_array in enumerate(args_dict["layout_array_name"])
+        ]
+
+    for one_file in telescope_file:
+
+        logger.debug(f"Processing: {one_file}.")
+        for one_angle in rotate_angles:
+            logger.debug(f"Processing: {one_angle}.")
+            base_name = (Path(one_file).name).split(".")[0] + "_"
+            if args_dict["figure_name"] is None:
+                print(one_angle.to(u.deg))
+                plot_file_name = (
+                    "plot_layout_array_"
+                    + base_name
+                    + str((round((one_angle.to(u.deg).value))))
+                    + "deg"
+                )
+            else:
+                plot_file_name = args_dict["figure_name"]
+
+            telescope_table = LayoutArray.read_telescope_list_file(one_file)
+            telescopes_dict = LayoutArray.include_radius_into_telescope_table(telescope_table)
+            fig_out = plot_array(
+                telescopes_dict, rotate_angle=one_angle, show_tel_label=args_dict["show_tel_label"]
+            )
+            output_dir = io_handler_instance.get_output_directory(
+                label, dir_type="application-plots"
+            )
+            plot_file = output_dir.joinpath(plot_file_name)
+
+            for f in ["pdf", "png"]:
```

Comment:
I modified this to be `ext`, but it's not important.

Reply:
again, I dont see your commit. please remember to checkout the branch after my changes in case you want to change directly something in the code

Reply:
Again, forgot to push. This resulted in a conflict so I reimplemented the small change after your modifications. Your modifications are a bit more complex than I expected though, so I will review that now.